colibre/scripts/stellar_abundances_FeH_MgFe.py

```diff
@@ -1,13 +1,13 @@
 """
-Plots the stellar abundances of given snapshot
+Plots the stellar abundances ([Fe/H] vs [Mg/Fe]) for a given snapshot
 """
 import matplotlib.pyplot as plt
 import numpy as np
 import unyt
 import glob
 from swiftsimio import load
 from swiftpipeline.argumentparser import ScriptArgumentParser
-from velociraptor.observations import load_observation
+from velociraptor.observations import load_observations
```

Comment:
`load_observation` has a deprecation warning; using `load_observations `is preferred.